contracts/contracts/open_gsn/UmbraPaymaster.sol

```diff
@@ -14,7 +14,7 @@ contract UmbraPaymaster is BasePaymaster {
   }
 
   function versionPaymaster() external view override returns (string memory) {
-    return "1.0.0";
+    return "2.0.0";
```

Comment:
Do we actually need to bump this to 2.0.0 given that it's never been used for anything real? It looks like this method is not used in signatures anywhere so it shouldn't affect replay attacks

Reply:
Good question. I'm not exactly sure what was causing it, but something was throwing an error until I updated this value. I think perhaps their JS library (used in the tests) for interacting with the contracts will throw if the versions don't agree.